doc/api/quic.md

```diff
@@ -212,7 +212,7 @@ session.on('stream', (stream) => {
 
 #### QuicStream headers
 
-Some QUIC application protocols (like HTTP/3) make use of headers.
+Some QUIC application protocols (like HTTP/3) use headers.
```

Comment:
Please put the changes to quic.md into a separate commit to make it easier to backport the rest of the changes.

Reply:
OK, I've split the quic.md change into its own commit.